ext/spl/spl_fixedarray.stub.php

```diff
@@ -49,18 +49,6 @@ public function offsetSet($index, $value) {}
      */
     public function offsetUnset($index) {}
 
-    /** @return void */
-    public function rewind() {}
-
-    /** @return mixed */
-    public function current() {}
-
-    /** @return int */
-    public function key() {}
-
-    /** @return void */
-    public function next() {}
-
-    /** @return bool */
-    public function valid() {}
+    /** @return Iterator */
```

Comment:
Should update the `implements` line above as well.

Reply:
Also, since this is a new method, you can use `: Iterator`.

Reply:
> Should update the `implements` line above as well.

Updating, thank you for the catch!

Reply:
> Also, since this is a new method, you can use `: Iterator`.

Uhh... not very sure about this one. Is there a different notation for new methods?

Reply:
Thanks. Updating this one as well.